rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -440,6 +480,40 @@ pub async fn install_nns_canisters(
     )
     .await;
 
+    if install_index_canister {
```

Comment:
It is nice that you took care to keep the public interface behavior unchanged using thse flags, but does this code need to be included in this function at all. I mean, could this be another separate function?

Reply:
Do you suggest extracting the code that deploys `index` and `cycles_minting` canisters into separate functions?

Reply:
The idea behind introducing `install_nns_suite` is to be able to deploy the entire NNS stack via the single helper and since the index canister doesn't make a lot of sense without the ledger canister, it's easier to install them together

Reply:
> it's easier to install them together..

What I meant was another function that installed them together. Not two separate functions for each. Sorry for the confusion.

Reply:
Ah, something like `install_all_nns_canisters` that will use `true` for both of these boolean flags, right?

Reply:
> Ah, something like install_all_nns_canisters that will use true for both of these boolean flags, right?

I was thinking of a function that only install these two canisters. So that would leave the original `install_nns_canisters` as it was before.  But it seems that to do that, we should move the building of `nns_init_payload_builder` out of it as well. But I am not sure how much of the builder built in the function is actually used in the initialization of the new canisters. 

If that is possible, I think it might be a little more cleaner, even though it is just minimizing internal noise.

Reply:
> I was thinking of a function that only install these two canisters

These two canisters don't make a lot of sense without the rest of the NNS canisters.

> But it seems that to do that, we should move the building of `nns_init_payload_builder` out of it as well

Yeah, this was one of the culprits too